CHANGELOG.md

```diff
@@ -2,4 +2,5 @@
 
 ## 1.0.0 August 26, 2020
 
-- Initial release.
+- Bug #41: Fix bug with instances of extended `DateTime` class (@Tigrov)
+- Enh #41: Improve performance (@Tigrov)
```

Comment:
These lines should be in the new `1.0.1` section.

Reply:
Fixed. Thanks.